macros/query_comment.sql

```diff
@@ -23,6 +23,7 @@
             node_resource_type=node.resource_type,
             node_meta=node.config.meta,
             node_tags=node.tags,
+            run_started_at=run_started_at.strftime("%Y-%m-%d %H:%M:%S"),
```

Comment:
Wondering if we can go ISO 8601 UTC here?

Reply:
Hey @NiallRees!! I've just updated and tested:
![image](https://github.com/get-select/dbt-snowflake-query-tags/assets/151600803/9390a9b9-4126-416a-9c88-77a0308b4007)